lib/src/Uncertainty/Algorithm/OrthogonalBasis/OrthogonalProductFunctionFactory.cxx

```diff
@@ -99,13 +93,17 @@ OrthogonalProductFunctionFactory::FunctionFamilyCollection OrthogonalProductFunc
   return coll;
 }
 
-
 /* Build the Function of the given index */
 Function OrthogonalProductFunctionFactory::build(const UnsignedInteger index) const
 {
   return tensorizedFunctionFactory_.build(index);
 }
 
+/* Build the Function of the given index */
+Function OrthogonalProductFunctionFactory::build(const Indices & indices) const
```

Comment:
```suggestion
Function OrthogonalProductFunctionFactory::buildFromMultiIndices(const Indices & indices) const
```
It should be clear that the purpose is _not_ to iterate over the indices. The goal is to invert the enumerate function, then build the single function associated with this index.

Reply:
No: `build(const UnsignedInteger index)` and `buildFromMultiIndices(const Indices & indices)` have the exact same purpose, and this is why they should have the same name. Only the input arguments are different, but the output is the same. I will not integrate that change.